### PR TITLE
Update Get-DtcDefault.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue - "fill in the blank" assignments - Description has been updated now.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @tseward

Peer review needed for this PR. Thanks.

**Comment by tseward** [comment]
Agree with @dariomws306, otherwise looks good.

File: docset/winserver2012r2-ps/msdtc/Get-DtcDefault.md [hunk 1]
```diff
@@ -45,7 +45,14 @@ This command queries the default DTC server name.
 ## PARAMETERS
 
 ### -AsJob
-{{Fill AsJob Description}}
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to complete. 
+
+The cmdlet immediately returns an object that represents the job and then displays the command prompt. 
+You can continue to work in the session while the job completes. 
+To manage the job, use the `*-Job` cmdlets. 
+To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/msdtc/Get-DtcDefault.md]:
```suggestion
To get the job results, use the [Receive-Job](https://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
```

File: docset/winserver2012r2-ps/msdtc/Get-DtcDefault.md [hunk 2]
```diff
@@ -45,7 +45,14 @@ This command queries the default DTC server name.
...
 ### -AsJob
-{{Fill AsJob Description}}
+Runs the cmdlet as a background job. Use this parameter to run commands that take a long time to complete. 
+
+The cmdlet immediately returns an object that represents the job and then displays the command prompt. 
+You can continue to work in the session while the job completes. 
+To manage the job, use the `*-Job` cmdlets. 
+To get the job results, use the [Receive-Job](http://go.microsoft.com/fwlink/?LinkID=113372) cmdlet. 
+
+For more information about Windows PowerShell background jobs, see [about_Jobs](http://go.microsoft.com/fwlink/?LinkID=113251).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/msdtc/Get-DtcDefault.md]:
```suggestion
For more information about Windows PowerShell background jobs, see [about_Jobs](https://go.microsoft.com/fwlink/?LinkID=113251).
```

**Comment by v-raaba** [comment]
@e0i @sowjanya365 Closing this PR; please PR against Master

**Pull request closed without merging**